src/Engines/QueryBuilderEngine.php

```diff
@@ -102,20 +102,33 @@ public function filtering()
     {
         $this->query->where(
             function ($query) {
-                $keyword = $this->setupKeyword($this->request->keyword());
+                $globalKeyword = $this->setupKeyword($this->request->keyword());
+                $queryBuilder = $this->getQueryBuilder($query);
+
                 foreach ($this->request->searchableColumnIndex() as $index) {
                     $columnName = $this->getColumnName($index);
-
+                    // check if custom column filtering is applied
                     if (isset($this->columnDef['filter'][$columnName])) {
-                        $method     = Helper::getOrMethod($this->columnDef['filter'][$columnName]['method']);
-                        $parameters = $this->columnDef['filter'][$columnName]['parameters'];
-                        $this->compileColumnQuery(
-                            $this->getQueryBuilder($query),
-                            $method,
-                            $parameters,
-                            $columnName,
-                            $keyword
-                        );
+                        $columnDef = $this->columnDef['filter'][$columnName];
+                        // check if global search should be applied for the specific column
+                        $applyGlobalSearch = count($columnDef['parameters']) == 0 || end($columnDef['parameters']) !== false;
```

Comment:
Yes, but $applyGlobalSearch was done intentionally, so I could skip LIKE part for the particular column, when defining a custom filter. Most of the time you might not want to apply global filter for a custom filter column, even it would work with an added "or" but there might be cases where it becomes difficult to do an exact match. I would recommend to leave this option as is for the user to decide and be more flexible on defining the flow of custom filtering.

usage example code, when the applyGlobalSearch is not applied (this does guarantee an exact match for the column):
```php
      ->filterColumn('id', function($query, $keyword) {
             $query->where('id', $keyword);
       }, false)  // override a global search (disable) for the custom filter 
```

Cheers!